src/AzurePipelinesToGitHubActionsConverter.Core/PipelinesToActionsConversion/GeneralProcessing.cs

```diff
@@ -86,17 +86,17 @@ public AzurePipelines.Environment ProcessEnvironmentV2(string environmentYaml)
                             string resourceName = null;
                             if (json["resourceName"] != null)
                             {
-                                name = json["resourceName"].ToString();
+                                resourceName = json["resourceName"].ToString();
```

Comment:
avoid 'magic strings', 'resourceName = json[nameof(resourceName).ToString();'

Reply:
I don't know what 'magic strings' means. Please explain.

Reply:
Anything inside quotes is outside the scope of the compiler, therefore when parameters/arguments/etc get renamed then you get instances where fieldOldName = "fieldNewName" happen which can make code harder to follow/debug.
Basically removing magic strings leads to tighter coupling between code and API.
https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings

Reply:
Ahh got it. So use CONST instead. I'll think about it. Migration apps are a bit different than a regular app and resource strings/etc. It's a lot of work in this app (I have a lot of magic strings now that I'm parsing the JSON), for something that doesn't add much value, in my opinion.  I do appreciate the idea though. I'll keep it in mind.